### DISCUSSION TITLE
Benefits of Github student dev pack

### DISCUSSION THREAD

**Discussion by ImanBanerjee** [community/community#76816] [GitHub Education]
### Hi everyone,

is there any other benefit that comes with the normal pack features like discounts trials and stuff?

**Comment by kanduru-abhiram** [comment]
Hey @ImanBanerjee, there are many benefits that is covered under Student Developer pack, do check out below link for a detailed list of all the benefits covered under the Dev pack.

https://education.github.com/pack/offers

Please accept this as a solution of helped, thanks
